File: repository-local/src/main/java/com/andyanika/translator/repository/local/DatabaseTranslator.java
package com.andyanika.translator.repository.local;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.andyanika.translator.repository.local.model.FavoriteModel;
import com.andyanika.translator.repository.local.model.WordModel;

@Database(version = 1,
        entities = {
                WordModel.class,
                FavoriteModel.class})
abstract class DatabaseTranslator extends RoomDatabase {
    public abstract TranslatorDao translatorDao();
}

File: repository-local/src/main/java/com/andyanika/translator/repository/local/DatabaseTranslator.kt
package com.andyanika.translator.repository.local

import androidx.room.Database
import com.andyanika.translator.repository.local.entity.WordEntity
import androidx.room.RoomDatabase
import com.andyanika.translator.repository.local.entity.FavoriteEntity

@Database(version = 1, entities = [WordEntity::class, FavoriteEntity::class])
internal abstract class DatabaseTranslator : RoomDatabase() {
    abstract fun translatorDao(): TranslatorDao
}
